src/styles/main.scss

```diff
@@ -1,3 +1,69 @@
-body {
+$size: 100px;
+$gap: 1px;
+$day-of-week:
+  mon 0,
+  tue 1,
+  wed 2,
+  thu 3,
+  fri 4,
+  sat 5,
+  sun 6;
+
+* {
+  box-sizing: border-box;
+}
+
+.body {
+  font-family: Arial, Helvetica, sans-serif;
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  max-width: $size * 7 + $gap * 10;
```

Comment:
You can transfer mathematical operations to separate variables and use the result immediately, this will increase readability.